### ISSUE TITLE
Update gf-rgl build instructions

### ISSUE THREAD

**Issue by fredefox** [MUSTE-Project/MULLE#85]
Peter, could you help me update the gf-rgl build instructions. It's not quite working for me. If you give me precise instructions on how to do it from a fresh machine I can even add it to the custom-setup I have just introduced (#84).

See my comment in 

https://github.com/MUSTE-Project/MULLE/blob/bae9db56ab7ba3a2cee2f8cbc8ec611786807a05/README.md#L39-L45

**Comment by heatherleaf** [comment]
I don't think you should make or install the submodule RGL, because then it will try to install all compiled `.gfo` files in the `$GF_LIB_PATH` directory.

Instead you shouldn't have to do anything, only run make install from the top directory (i.e., the main MULLE Makefile which in turn calls the makefiles for the grammars and copies them to the `~ /.local/share/` directory.

**Comment by heatherleaf** [comment]
I updated the README by simply removing all instructions to install the RGL. It should work anyway, I hope.

(This requires that the soft-links in `novo_modo` and `exemplum` are correct. Do you know if they work on Windows systems too?)

**Comment by fredefox** [comment]
Git shims soft-links on Windows. So this should work. There might be other problems with installing on Windows that we don't know about however...

**Comment by fredefox** [comment]
> Instead you shouldn't have to do anything, only run make install from the top directory (i.e., the main MULLE Makefile which in turn calls the makefiles for the grammars and copies them to the ~ /.local/share/ directory.

This is not needed any more. I added [a custom setup script](https://github.com/MUSTE-Project/MULLE/blob/fredefox/muste/Setup.hs) that does this.

**Comment by heatherleaf** [comment]
Doesn't the following subsection in the README have to be updated too (lines 100+), after your changes in d978c22d8794873748cd09fca30337cf053f83cf?

```
### Building

The grammar-files (in `muste/data/grammars/`) must be kept up-to-date
and installed in a global location for the application to work
correctly.  This should be handled automatically by the cabal build
script.  If this somehow goes wrong you can do it automatically using:

    make install

Assuming you have set up `gf-core` and `gf-rgl`.

    stack build

The front end dependencies are managed with `npm`.  To fetch all
dependencies navigate to.

    muste-ajax/static/

And run

    npm install

To get fetch all dependencies.
```

**Comment by fredefox** [comment]
Well... It's not strictly necessary to do this, but it can't hurt either. I just figured it served as a good form of documentation for what's going on.

**Comment by heatherleaf** [comment]
Good point, perhaps the just add a sentence saying that this is what `stack install` does?

**Comment by fredefox** [comment]
Closed in 754ce67
